documentation/modules/proc-iot-configuring-sigfox.adoc

```diff
@@ -0,0 +1,143 @@
+// Module included in the following assemblies:
+//
+// assembly-iot-guide.adoc
+// assembly-IoT.adoc
+
+ifeval::["{cmdcli}" == "oc"]
+:cmd-get-adapter: echo "https://$(oc -n enmasse-infra get routes iot-sigfox-adapter --template='{{ .spec.host }}')"
+endif::[]
+ifeval::["{cmdcli}" != "oc"]
```

Comment:
I had this in the beginning, however this failed to render properly in the Atom editor preview. Because `cmdcli` was not set in the preview, it was neither `oc` nor `kubectl`. This way I had the Kubernetes instructions in the preview. Is there a better way to handle this?

Reply:
@ctron, ah, got it. That is one downside with Atom Preview. One thing that you can do is do a local build:
1. cd documentation
2. make publish

Reply:
Hm ok … if that is the way to do it. Then I will revert my change here.